packages/pods-core/src/builtin/NotionPod.ts

```diff
@@ -0,0 +1,156 @@
+import { DendronError, ERROR_SEVERITY, NoteProps } from "@dendronhq/common-all";
+import { markdownToBlocks } from "@instantish/martian";
+import type {
+  Page,
+  TitlePropertyValue,
+} from "@notionhq/client/build/src/api-types";
+import { Client } from "@notionhq/client";
+import _ from "lodash";
+import { ExportPod, ExportPodPlantOpts, ExportPodConfig } from "../basev3";
+import { JSONSchemaType } from "ajv";
+import { NotionUtilMethods, PodUtils } from "../utils";
+import { RateLimiter } from "limiter";
+
+const ID = "dendron.notion";
+
+// Allow 3 req/sec. Also understands 'hour', 'minute', 'day', or a no. of ms
+// @ts-ignore
+const limiter = new RateLimiter({ tokensPerInterval: 3, interval: "second" });
+
+type NotionExportPodCustomOpts = {
+  apiKey: string;
+  vault: string;
+};
+
+export type NotionExportConfig = ExportPodConfig & NotionExportPodCustomOpts;
+
+export class NotionExportPod extends ExportPod<NotionExportConfig> {
+  static id: string = ID;
+  static description: string = "export notes to notion";
+
+  get config(): JSONSchemaType<NotionExportConfig> {
+    return PodUtils.createExportConfig({
+      required: ["apiKey", "vault"],
+      properties: {
+        apiKey: {
+          type: "string",
+          description: "Api key for Notion",
+        },
+        vault: {
+          type: "string",
+          description: "vault to export from",
+        },
+      },
+    }) as JSONSchemaType<NotionExportConfig>;
+  }
+
+  /**
+   * Method to create pages in Notion
+   */
+  createPagesInNotion = async (blockPagesArray: any, notion: Client) => {
+    // rate limiter method
+    const sendRequest = async () => {
+      blockPagesArray.forEach(async (block: any) => {
+        // @ts-ignore
+        await limiter.removeTokens(1);
+        try {
+          await notion.pages.create(block);
+        } catch (error) {
+          this.L.error({
```

Comment:
It's not necessary to log this error, as it'll get logged with the throw below.  You can pack the msg here into the DendronError below and just throw that.  

Otherwise, we'll get a double error report in the error reporting backend.